diff --git a/node_modules/@novu/node/build/main/lib/novu.js b/node_modules/@novu/node/build/main/lib/novu.js
index e932181..feec37e 100644
--- a/node_modules/@novu/node/build/main/lib/novu.js
+++ b/node_modules/@novu/node/build/main/lib/novu.js
@@ -29,6 +29,7 @@ var environments_1 = require("./environments/environments");
 var feeds_1 = require("./feeds/feeds");
 var topics_1 = require("./topics/topics");
 var integrations_1 = require("./integrations/integrations");
+var fetchAdapter = __importDefault(require("@vespaiach/axios-fetch-adapter"));
 var Novu = (function (_super) {
     __extends(Novu, _super);
     function Novu(apiKey, config) {
@@ -39,6 +40,7 @@ var Novu = (function (_super) {
             headers: {
                 Authorization: "ApiKey " + _this.apiKey,
             },
+            adapter: fetchAdapter,
         });
         _this.subscribers = new subscribers_1.Subscribers(_this.http);
         _this.environments = new environments_1.Environments(_this.http);
diff --git a/node_modules/@novu/node/build/module/lib/novu.js b/node_modules/@novu/node/build/module/lib/novu.js
index f4685e5..be5e02f 100644
--- a/node_modules/@novu/node/build/module/lib/novu.js
+++ b/node_modules/@novu/node/build/module/lib/novu.js
@@ -10,6 +10,7 @@ import { Environments } from './environments/environments';
 import { Feeds } from './feeds/feeds';
 import { Topics } from './topics/topics';
 import { Integrations } from './integrations/integrations';
+import fetchAdapter from '@vespaiach/axios-fetch-adapter';
 export class Novu extends EventEmitter {
     constructor(apiKey, config) {
         super();
@@ -19,6 +20,7 @@ export class Novu extends EventEmitter {
             headers: {
                 Authorization: `ApiKey ${this.apiKey}`,
             },
+            adapter: fetchAdapter,
         });
         this.subscribers = new Subscribers(this.http);
         this.environments = new Environments(this.http);
