diff --git a/node_modules/@remix-run/dev/dist/config.d.ts b/node_modules/@remix-run/dev/dist/config.d.ts
index cada1fa..708d714 100644
--- a/node_modules/@remix-run/dev/dist/config.d.ts
+++ b/node_modules/@remix-run/dev/dist/config.d.ts
@@ -7,7 +7,7 @@ export interface RemixMdxConfig {
 export declare type RemixMdxConfigFunction = (filename: string) => Promise<RemixMdxConfig | undefined> | RemixMdxConfig | undefined;
 export declare type ServerBuildTarget = "node-cjs" | "arc" | "netlify" | "vercel" | "cloudflare-pages" | "cloudflare-workers" | "deno";
 export declare type ServerModuleFormat = "esm" | "cjs";
-export declare type ServerPlatform = "node" | "neutral";
+export declare type ServerPlatform = "node" | "neutral" | "browser";
 declare type Dev = {
     port?: number;
     appServerPort?: number;
diff --git a/node_modules/@remix-run/dev/dist/config.js b/node_modules/@remix-run/dev/dist/config.js
index c0667a4..920a2a9 100644
--- a/node_modules/@remix-run/dev/dist/config.js
+++ b/node_modules/@remix-run/dev/dist/config.js
@@ -90,7 +90,7 @@ async function readConfig(remixRoot, serverMode = serverModes.ServerMode.Product
     case "cloudflare-workers":
     case "deno":
       serverModuleFormat = "esm";
-      serverPlatform = "neutral";
+      serverPlatform = "browser";
       break;
   }
   let mdx = appConfig.mdx;
