diff --git a/node_modules/@remix-run/dev/dist/config.d.ts b/node_modules/@remix-run/dev/dist/config.d.ts
index 30160bf..f3939b4 100644
--- a/node_modules/@remix-run/dev/dist/config.d.ts
+++ b/node_modules/@remix-run/dev/dist/config.d.ts
@@ -7,7 +7,7 @@ export interface RemixMdxConfig {
 export declare type RemixMdxConfigFunction = (filename: string) => Promise<RemixMdxConfig | undefined> | RemixMdxConfig | undefined;
 export declare type ServerBuildTarget = "node-cjs" | "arc" | "netlify" | "vercel" | "cloudflare-pages" | "cloudflare-workers" | "deno";
 export declare type ServerModuleFormat = "esm" | "cjs";
-export declare type ServerPlatform = "node" | "neutral";
+export declare type ServerPlatform = "node" | "neutral" | "browser";
 interface FutureConfig {
     v2_meta: boolean;
 }
diff --git a/node_modules/@remix-run/dev/dist/config.js b/node_modules/@remix-run/dev/dist/config.js
index 5b2b362..66ac968 100644
--- a/node_modules/@remix-run/dev/dist/config.js
+++ b/node_modules/@remix-run/dev/dist/config.js
@@ -89,7 +89,7 @@ async function readConfig(remixRoot, serverMode = serverModes.ServerMode.Product
     case "cloudflare-workers":
     case "deno":
       serverModuleFormat = "esm";
-      serverPlatform = "neutral";
+      serverPlatform = "browser";
       break;
   }
   let mdx = appConfig.mdx;
